e2e/cypress/e2e/kml.cy.js

```diff
@@ -0,0 +1,41 @@
+import {export_with_interaction, test_and_save_download, test_without_interaction} from './utils.js';
+
+before(() => {
+    cy.exec('rm -rf cypress/downloads/*');
+});
+
+
+it('detects server', () => {
+    cy.visit('/');
+    cy.get('.mat-button-wrapper').should('contain', 'So funktioniert es!');
+})
+
+
+it('Test small KML file with UI change', () => {
```

Comment:
I think lines 14-20 can be removed, as this KML file is also tested in the batch-test.